core/src/client/test_client.rs

```diff
@@ -136,7 +136,7 @@ impl TestBlockChainClient {
         let genesis_block = scheme.genesis_block();
         let genesis_header = scheme.genesis_header();
         let genesis_hash = genesis_header.hash();
-        let genesis_score = *genesis_header.score();
+        let genesis_score = U256::default();
```

Comment:
Who uses it? Why the score remains after this pr?

Reply:
I didn't completely remove the score from this PR. I discussed this work with @majecty  and decided not to remove the score from the Header struct and the Block struct and the fields related to these. It is to prevent conflicts with other team members currently using or modifying this part. According to the @majecty , changing the header and block field, the logic and e2e test in relation to it can cause a collision, so we don't have to do this right now. So I didn't change some fields, only removed the score that had nothing to do with the logic involved and removal.

Reply:
By the way, this logic `let genesis_score = *genesis_header.score()` seems unnecessary. It's my mistake.

Reply:
@sgkim126 I modified it, and all the unnecessary parts that were not removed were also removed. Thank you very much for your careful review. In that sense, could you check it one more time? : )))